readme.md

```diff
@@ -1,7 +1,7 @@
 # Frontend practice with catalog page
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
 - [DEMO LINK](https://<your_account>.github.io/layout_catalog/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_catalog/report/html_report/)
+- [TEST REPORT LINK](https://anton-chornobai.github.io/layout_catalog/report/html_report/)
```

Comment:
Also you need to fix the demo link
Add these links to the PR description
<img width="941" alt="image" src="https://github.com/mate-academy/layout_catalog/assets/64750958/2054ac71-fe6e-455e-85ed-996bdb0d8801">